bluetooth/index.md

```diff
@@ -46,7 +46,7 @@ custom developed for the micro:bit. Read about it
 
 [Pairing your Bluetooth device](https://support.microbit.org/solution/articles/19000051025-pairing-and-flashing-code-via-bluetooth)
 
-[Bluetooth Developer Studio](https://www.bluetooth.com/download-developer-studio)
+[Bluetooth Developer portal](https://www.bluetooth.com/develop-with-bluetooth/)
```

Comment:
It's a pity Bluetooth Developer Studio has been discontinued :(
I'm not sure if having this link offers anything for micro:bit users, seems like it is targeted for product developers.

Reply:
I've removed this link and the other that linked to adeveloper studio tutorial. I'd agree that if there's no context for the link it should be removed